base/shell/explorer/lang/it-IT.rc

```diff
@@ -187,7 +187,7 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_START "Avvio"
+    IDS_START "Start"
```

Comment:
"Start" doesn't sound italian?

Reply:
Yes but we used to call that Start since Windows 98 (and after Microsoft renamed it to start also in Italian language) so I suppose that si better not to translate that.